py3status/modules/vpn_status.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python3
+"""
+Drop-in replacement for i3status run_watch VPN module.
+
+Expands on the i3status module by displaying the name of the connected vpn
+using the NetworkManager python module.
+
+Configuration parameters:
+    cache_timeout: How often we refresh this module in seconds.
+        (default 10)
+    check_pid: If true, act just like the default i3status module.
+        (default False)
+    format: Format of the output.
+        (default 'VPN: {name}')
+    pidfile: Same as i3status.conf pidfile, checked when check_pid is True.
+        (default '/sys/class/net/vpn0/dev_id')
+
+Format string parameters:
+    {name} The name and/or status of the VPN.
+
+Requires:
+    python-NetworkManager
+
+@author Nathan Smith <nathan AT praisetopia.org>
+"""
+
+try:
+    import NetworkManager
```

Comment:
do we need `NetworkManager`?  Can we get the info just from dbus?

 ` dbus-monitor --system`  start/stop vpn